terraform/modules/infrastructure/secdevops/aws_ami.tf

```diff
@@ -10,15 +10,16 @@ data "aws_ami" "ubuntu" {
 }
 
 
-data "aws_ami" "debianstretch" {
-  most_recent       = true
-  owners            = ["679593333241"]
-
-  filter {
-    name   = "name"
-    values = ["debian-stretch-hvm-x86_64-*"]
-  }
-}
+//data "aws_ami" "debianstretch" {
+//  most_recent       = true
+//  owners            = ["679593333241"]
+//
```

Comment:
this needs to be reverted - it was commented out due to not being available in the me-south-1 region